Scripts/extras/pull_and_build_from_git.py

```diff
@@ -840,6 +858,12 @@ def prepare_build(platform_name, base_folder, build_folder, package_root_folder,
                             action="store_true")
         parser.add_argument('--build-path',
                             help="Path to build the repository in. Defaults to {base_path}/temp.")
+        parser.add_argument('--git-path',
+                            help='Path to an existing git repo',
+                            default=None)
+        parser.add_argument('--git-skip',
```

Comment:
See comment about this above. And git-path is ambigous, it should be '--local-repo'

Reply:
will do.